model/cluster_config.go

```diff
@@ -0,0 +1,7 @@
+package model
+
+type CluserConfig struct {
```

Comment:
Could it be that this is supposed to be `ClusterConfig`? Note that it will have to be changed in the other 14 places

Reply:
sure, will change this.